src/blueapi/worker/task_worker.py

```diff
@@ -95,7 +95,7 @@ def __init__(
 
         self._tasks = {}
 
-        self._state = WorkerState.from_bluesky_state(ctx.run_engine.state)
+        self._state = WorkerState.from_bluesky_state(ctx.run_engine.state)  # type: ignore
```

Comment:
This seems fixable by setting the return type of `from_bluesky_state`, possibly including a `cast`?

Reply:
will look into it

Reply:
outdated